app/controllers/volunteer_ops_controller.rb

```diff
@@ -97,11 +101,15 @@ def current_user_has_organisation?
   end
 
   def set_organisation
-    @organisation = Organisation.friendly.find(params[:organisation_id]) rescue nil
+    @organisation = Organisation.friendly.find(params[:organisation_id])
+    rescue ActiveRecord::RecordNotFound
+      @organisation = nil
```

Comment:
I refactored set_organisation method to satisfy "codeclimate" which had issue, "avoid using rescue in its modifier form". Note that rescuing from nil or ActiveRecord::RecordNotFound is probably not necessary as this controller did not make the "set_organisation" a "prepend_before_action" filter as OrganisationsController did. Thus the rescue code is not executed by any test, causing "coverage/coveralls" check to fail.